portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,49 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Mariah Crawford's Portfolio</h1>
+      <p>A Princeton University student pursuing a Bachelor of Science in Engineering (BSE) 
+          in Computer Science and certificates in Entrepreneurship and Theater. Passionate 
+          about using her creativity to explore the intersections of technology and entrepreneurship. 
+          Exhibits strong computer programming skills in Java and Python languages and is a highly 
+          motivated individual and a quick learner. Demonstrates great leadership skills through her 
+          involvement as an officer in the diSiac Dance Company, Student Government (USG), and the 
+          Entrepreneurship Club.</p>
+      <img src="images/me.jpg" style="width:700px;height:500px;"/>
+
+      <p>Click here to get a random fun fact about Mariah:</p>
+      <button onclick="addRandomGreeting()">Generate Fun Fact</button>
+      <div id="funfact-container"></div>
+
+      <h1>More Information:</h1>
+
+      <h2>Projects </h2>
+      <p>Software Product Print Portfolio Project, Sonorines Research Project</p>
+
+      <h2> External Links </h2>
+      <p>Click <a href="http://linkedin.com/in/mariah-crawford-38b296179">here</a> to visit my LinkedIn Page</p>
+      <p>Click <a href="https://github.com/mariah-crawford">here</a> to visit my GitHub Profile</p>
+
+      <h2>About Me: Hobbies and Interests</h2>
+      <p>I'm passionate about volunteering and offering guidance and leadership to younger students.
+          Through Student Government I am able to coordinate with other organizations to organize 
+          and participate in volunteer events. Also, I was a mentor to many incoming college freshmen as 
+          a leader in Princeton University's Freshmen Orienation Program, Community Action, in which we engaged
+          in community service while building a sense of community and belonging among the students.
+      </p>
+
+      <img src="images/volunteer.jpg" style="width:700px;height:500px;"/>
+      <img src="images/ca.jpg" style="width:700px;height:500px;"/>
```

Comment:
Line breaks in HTML files are super helpful for authors and code readers to quickly glance at different sections. You have done a great job here in separating each section with new lines. These img tags should be moved to the previous section about volunteering to make it easy for the code reader to associate these images with the previous paragraph.

Reply:
Awesome, thank you! I added a new commit to reorganize the volunteer section in the html file.